components/Relationships/Relationships.tsx

```diff
@@ -0,0 +1,69 @@
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import { useRelationships } from 'utils/api/relationships';
+import RelationshipElement from './RelationshipElement';
+
+interface Props {
+  id: number;
+}
+
+const Relationships = ({ id }: Props): React.ReactElement => {
+  const { data: { personalRelationships } = {}, error } = useRelationships(id);
+
+  if (!personalRelationships) {
+    return <Spinner />;
+  }
+  if (error) {
+    return <ErrorMessage />;
+  }
+
+  let elm_count = 0;
+
+  try {
+    elm_count =
+      personalRelationships[0].parents.length +
+      personalRelationships[0].children.length +
+      personalRelationships[0].siblings.length +
+      personalRelationships[0].other.length;
+  } catch (error) {
+    elm_count = 0;
+  }
+
+  return (
+    <div>
+      <div>
+        <div className="lbh-table-header">
+          <h3 className="govuk-fieldset__legend--m govuk-custom-text-color">
+            RELATIONSHIPS
```

Comment:
should be sentence case - screen readers will SHOUT this

Reply:
okay - we should update them all then. here's a task: https://hackney.atlassian.net/browse/SCT-341